src/rules/style.ts

```diff
@@ -32,7 +32,7 @@ export default {
   // Enforces basic format rules for JSDoc comments.
   'jsdoc-format': true,
   // Requires that a default import have the same name as the declaration it imports. Does nothing for anonymous default exports.
-  'match-default-export-name': true,
+  'match-default-export-name': false,
```

Comment:
We had this enabled to generate some feedback. It creates more noise than benefits.